trio/_core/_run.py

```diff
@@ -1524,12 +1647,12 @@ def current_time(self):
         return self.clock.current_time()
 
     @_public
-    def current_clock(self):
+    def current_clock(self) -> trio._abc.Clock:
         """Returns the current :class:`~trio.abc.Clock`."""
         return self.clock
 
     @_public
-    def current_root_task(self):
+    def current_root_task(self) -> trio.lowlevel.Task | None:
```

Comment:
Why `trio.lowlevel.Task` when `Task` is from this file

Reply:
It's because of `@_public` - that copies this definition to the `_generated_run.py` file, so by using the full name we only need `import trio` in that.